components/playlist/browser/resources/api/api.ts

```diff
@@ -24,7 +24,7 @@ class API {
   #pageHandler = new PlaylistServiceRemote()
   #nativeUI = new PlaylistNativeUIRemote()
 
-  constructor () {
+  constructor() {
```

Comment:
This happens to me all the time too - I'm not even sure what our Brave style is meant to be :laughing:

Reply:
Yeah, makes it difficult to review changes 😥